src/addons/metaservice/src/main/java/ee/ria/xroad/proxy/serverproxy/MetadataServiceHandlerImpl.java

```diff
@@ -120,6 +120,16 @@ private static SAXTransformerFactory createSaxTransformerFactory() {
         try {
             SAXTransformerFactory factory = (SAXTransformerFactory) TransformerFactory.newInstance();
             factory.setFeature(XMLConstants.FEATURE_SECURE_PROCESSING, true);
+            try {
+                factory.setAttribute(XMLConstants.ACCESS_EXTERNAL_DTD, "");
```

Comment:
Can you elaborate a bit more why this was required?

Reply:
to resolve this one https://github.com/nordic-institute/X-Road/security/code-scanning/23
from what I saw this  factory get's to deal with WSDL XMLs that comes from external systems so it may be ok add suggested changes